packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -190,6 +190,30 @@ type DatePickerProps = {
    * @default false
    */
   showYearSelection?: boolean;
+  /**
+   * Allows props to be passed for calendar month year.
```

Comment:
might be a good idea to improve the jsdocs to describe in slightly more detail which DOM element each of these props is referring to. it's not obvious from the current names and descriptions

Reply:
the `iui-` classes are an implementation detail so not useful in the description.

i meant more like describing in words what part specifically. for example: "Allows props to be passed for calendar month year, referring to the div that wraps around the month/year and the next/previous buttons"